### ISSUE TITLE
plot(argo, which=4) error on lacking lon and lat

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1325]
Doing e.g.
```R
u <- "http://www.usgodae.org/ftp/outgoing/argo/dac/meds/4901783/4901783_prof.nc"
f <-  gsub("^.*/(.*)$", "\\1", u)
if (0 == length(list.files(pattern=u)))
    download.file(u, f)

argo <- read.oce(f)
plot(argo, which=4)
```
yields
```R
Error in x[["SA"]] : 
  object lacks location information, so SA cannot be computed
> 
```

**Workaround** Use
```R
plotTS(argo)
```
instead.

**Comment by richardsc** [comment]
Seeing as it's trying to do `SA`, you must have `oceEOS="gsw"` in your options.

I tried your code and get a normal TS plot (without error, but with UNESCO variables).

**Comment by richardsc** [comment]
That being said, it's clearly an error because the `argo` object *does* have location information.

**Comment by dankelley** [comment]
Yup. I'll fix this up tomorrow most likely -- it's quite minor.  Also I just made a new issue about "adorn", which you can veto me on, if you like...

**Comment by dankelley** [comment]
My work and home machines have different EOS setups, as a way to check that oce works ok in both.

**Comment by dankelley** [comment]
Fixed in "develop" commit 8da161129ab0d539adcfb3fe8f2a6d4070eb9189 ... the hardest part was that my code fix used "longidude" and I wasn't seeing the typo in the error message. Guess I'm too much of a d00d.
